test/unit/org/apache/cassandra/index/sai/cql/intersection/RandomIntersectionTester.java

```diff
@@ -80,7 +80,8 @@ protected void runRestrictedQueries() throws Throwable
         Map<Integer, List<TestRow>> testRowMap = buildAndLoadTestRows();
 
         beforeAndAfterFlush(() -> {
-            for (int queryCount = 0; queryCount < nextInt(10, 100); queryCount++)
+            int queryCount = nextInt(10, 80);
```

Comment:
Why 10-80 instead of the previous 10-100?

Reply:
So I moved this out of the loop condition because it was re-evaluating on every iteration rather than being set before the loop, as I'm sure we originally intended. I think `10-80` ended up producing a distribution pretty similar to what was there before. (i.e. We very rarely ever got to the upper end of the distribution with re-evaluating on every iteration.)